ECE496-663/SH4RE
# [SH4RE-111] - Email Verification

**Verify email demo:** https://streamable.com/3eb3c2
**Unverified demo:** https://streamable.com/vqbm5t

### To test:
1. Create a new account with an email you have access to (you can use `+X` before the `@` to make more test emails, change `X` to a number)
2. Your email will be unverified, but you will have an email sent to you.
3. Click the link.
4. Then click the `I have already verified my email` button in the app.
5. It will make you log in again and you will be a verified user.
6. You should not be able to message users, post items or see messages.
